bcipy/language/README.md

```diff
@@ -12,3 +12,5 @@ The core methods of any `LanguageModel` include:
 
 You may of course define other methods, however all integrated BciPy experiments using your model will require those to be defined!
 
+The pretrained GPT2 language model is saved in [this folder on Google Drive](https://drive.google.com/drive/folders/1pkvwHA8SR7awxf7fj7Ds4FhY6SGxeGtX?usp=sharing). Download the files in the folder and put them in a local directory. Then use the path to the local directory to load the model. (Alternatively, just pass in the model name like "gpt2" as the language model path and the pretrained language model will be downloaded and stoed in local cache)
+
```

Comment:
Since BciPy is publicly available and not just used by our group we may not want to include the save model location in the README. We would need to commit to continuing to provide it at that location.

Reply:
I'd be okay removing the link and having them download as it describes in the alternate. For our own internal use cases, we can probably use the drive one. Not sure which would download faster.